### ISSUE TITLE
Suggestions 

### ISSUE THREAD

**Issue by ghost** [peter-iakovlev/Telegram#184]
Hello.

I have a portion of ideas for Telegram (both mobile and desktop). It's not and issues and I'm not sure where to post them. Can you enlighten me please?

**Comment by ghost** [comment]
And it always was interesting, how Telegram clients is developing? Am I right, that macOS and iOS versions leads by side developers, who just realize requested by main team features?
Correct me if I wrong.

Thank you.

**Comment by hawkins** [comment]
You're right, this is a third-party effort. Telegram's iOS users get shafted :( the MacOS and android support is much better

And I'm just a user here like you, but you can open an 'issue' for each suggestion, I think. That's usually how its done on projects like these

**Comment by ghost** [comment]
@hawkins Thanks.

Remarkable, that VK (also made by Durov) have much worse version for iOS than for Android, but after he's "leaving"...
